source/index.ts

```diff
@@ -267,9 +269,17 @@ class Conf<T extends Record<string, any> = Record<string, unknown>> implements I
 
 	/**
 	Delete all items.
+
+	This resets known items to their default values, if defined by the `defaults` or `schema` option.
 	*/
 	clear(): void {
+		const keys = Object.keys(this.store);
```

Comment:
If the current value of the store has a key deleted that is in the defaults or schema, I believe it wouldn't be included in this and you'd end up with an incomplete default.

Reply:
Good catch: https://github.com/sindresorhus/conf/pull/136/commits/8b1ee0b6eb2f5ff4226baa23dac3b1e44b77cf52